modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteDiscoDataHandlingInNewClusterTest.java

```diff
@@ -116,14 +130,26 @@
             );
         }
 
-        CacheConfiguration staticCacheCfg = new CacheConfiguration(STATIC_CACHE_NAME)
+        if (igniteInstanceName.equals(NODE_4_DIFF_STATIC_CONFIG) || igniteInstanceName.equals("clientWithDiffConfig"))
```

Comment:
Missed constant variable name usage "clientWithDiffConfig".

Reply:
I introduced the constant but didn't use it here. Fixed.